File: fix_quotes.js
const fs = require('fs');

function fixFile(path) {
    let content = fs.readFileSync(path, 'utf8');
    content = content.replace(/replace\(\/'\/g, "\\\\'"\)/g, 'replace(/&#39;/g, "\\\\&#39;")');
    fs.writeFileSync(path, content, 'utf8');
}

fixFile('app.js');
try {
    fixFile('beta/app.js');
} catch (e) {}

console.log("Fixed files");

File: fix_quotes.py
import sys

def fix_file(path):
    with open(path, 'r', encoding='utf-8') as f:
        content = f.read()
    content = content.replace("replace(/'/g, \"\\\\'\")", "replace(/&#39;/g, \"\\\\&#39;\")")
    with open(path, 'w', encoding='utf-8') as f:
        f.write(content)

fix_file('app.js')
fix_file('beta/app.js')
print("Fixed files")
